### PR TITLE
LPD-30188 Moved poshi test to playwright

### PR DESCRIPTION
**Pull request by alessiorendina**
https://liferay.atlassian.net/browse/LPD-30188

### CODE REVIEW

**Comment by alessiorendina** [comment]
ci:test:sf

File: modules/test/playwright/pages/commerce/commerce-product-content-web/productPublisherPage.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+	readonly removeTagNameButton: (tagName: string) => Promise<Locator>;
+	readonly tagsInput: Locator;
+
+	constructor(page: Page) {
+		this.closeButton = page.getByLabel('close', {exact: true});
+		this.configurationFilterLink = page
+			.frameLocator(
+				'iframe[title*="Product Publisher"][title*="Configuration"]'
+			)
+			.getByRole('link', {exact: true, name: 'Filter'});
```
Comment (smottal) [inline modules/test/playwright/pages/commerce/commerce-product-content-web/productPublisherPage.ts:30]: nice to have: this should be 
```
this.configurationFrame.getByRole...
```
but you have to save it after the other variable

**Review by smottal — APPROVED** [review]

File: modules/test/playwright/pages/commerce/commerce-product-content-web/productPublisherPage.ts (hunk 1 above)
Comment (alessiorendina) [reply modules/test/playwright/pages/commerce/commerce-product-content-web/productPublisherPage.ts:30]: I did like this to maintain the alphabetical order

**Comment by alessiorendina** [comment]
ci:test:playwright

**Comment by alessiorendina** [comment]
ci:test:playwright

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**